drivers/hmis_csv_twenty_twenty_two/app/models/hmis_csv_twenty_twenty_two/exporter/project_coc.rb

```diff
@@ -11,6 +11,10 @@ class ProjectCoc < GrdaWarehouse::Hud::ProjectCoc
 
     belongs_to :project_with_deleted, class_name: 'GrdaWarehouse::Hud::WithDeleted::Project', primary_key: [:ProjectID, :data_source_id], foreign_key: [:ProjectID, :data_source_id], inverse_of: :project_cocs, optional: true
 
+    scope :for_cocs, ->(coc_codes) do
+      where(CoCCode: coc_codes)
```

Comment:
This needs to look at potential overrides.  I believe the column is hud_coc_code.  Unlike computed_project_type, you'll need to check for `hud_coc_code is null and CoCCode = coc_codes or hud_coc_code = coc_codes`